packages/tables/resources/views/index.blade.php

```diff
@@ -822,10 +826,13 @@ class="text-sm font-semibold text-gray-950 dark:text-white"
                                     \Filament\Support\prepare_inherited_attributes($column->getExtraHeaderAttributeBag())
                                         ->class([
                                             'fi-table-header-cell-' . str($column->getName())->camel()->kebab(),
-                                            'w-full' => $column->canGrow(default: false),
+                                            'w-full' => blank($columnWidth) && $column->canGrow(default: false),
                                             '[&:not(:first-of-type)]:border-s [&:not(:last-of-type)]:border-e border-gray-200 dark:border-white/5' => $column->getGroup(),
                                             $getHiddenClasses($column),
                                         ])
+                                        ->style([
+                                            'width' => $columnWidth,
```

Comment:
@danharrin Thank you for your great effort, correct me if am wrong. Shouldn't this line (for example) be link this
                        'width:' .  $columnWidth => $columnWidth != null,

Your Advise is highly appreciated.

Reply:
I've adjusted it